schema/deploy/tables/attachment.sql

```diff
@@ -12,8 +12,7 @@ create table ggircs_portal.attachment
   file_size varchar(100),
   application_id integer not null references ggircs_portal.application(id),
   version_number int,
-  foreign key (application_id, version_number) references ggircs_portal.application_revision(application_id, version_number),
-  unique (application_id, version_number)
+  foreign key (application_id, version_number) references ggircs_portal.application_revision(application_id, version_number)
```

Comment:
This prevented people from uploading more than one attachment, ok to drop? It comes with the side effect that people can upload attachments with the same filename

Reply:
yes, this should be fine